test/nested-properties.test.js

```diff
@@ -0,0 +1,70 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback-datasource-juggler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// This test written in mocha+should.js
+'use strict';
+
+/* global getSchema:false, connectorCapabilities:false */
+var should = require('./init.js');
+
+var db, Model;
+
+describe('basic-querying', function() {
+  before(function(done) {
+    var modelDef = {
+      'nested': {
+        'duration': Number,
+        'happenedOn': Date,
+      },
+    };
+
+    db = getSchema();
+    Model = db.define('Model', modelDef);
+    db.automigrate(done);
+  });
+
+  describe('ping', function() {
```

Comment:
I am pretty sure `ping` is already tested elsewhere, there is no need to test it again in this file. Please remove this `describe` block.